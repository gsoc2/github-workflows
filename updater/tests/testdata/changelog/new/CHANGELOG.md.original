# Changelog

## 0.14.0

### Dependencies

- Capture `Debug.LogError()` and `Debug.LogException()` also on background threads ([#673](https://github.com/gsoc2/gsoc2-unity/pull/673))
- Adding override for Gsoc2 CLI URL ([#666](https://github.com/gsoc2/gsoc2-unity/pull/666))
