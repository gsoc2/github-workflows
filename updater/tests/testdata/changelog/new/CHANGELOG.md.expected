# Changelog

## Unreleased

### Dependencies

- Bump Dependency from v7.16.0 to v7.17.0 ([#123](https://github.com/gsoc2/dependant/pulls/123))
  - [changelog](https://github.com/gsoc2/dependency/blob/main/CHANGELOG.md#7170)
  - [diff](https://github.com/gsoc2/dependency/compare/7.16.0...7.17.0)

## 0.14.0

### Dependencies

- Capture `Debug.LogError()` and `Debug.LogException()` also on background threads ([#673](https://github.com/gsoc2/gsoc2-unity/pull/673))
- Adding override for Gsoc2 CLI URL ([#666](https://github.com/gsoc2/gsoc2-unity/pull/666))
